sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/javaCode.scala

```diff
@@ -143,10 +143,7 @@ trait Block extends TreeNode[Block] with JavaCode {
     case _ => code.trim
   }
 
-  def length: Int = {
-    // Returns a code length without comments
```

Comment:
Ur, one more comment; could you leave some comments here about why the current logic returns a length with comments?

Reply:
Ok, added.

Reply:
Thanks